### PR TITLE
Merge orbit branch to main

### PR DESCRIPTION
**Pull request by jgao222**
Most of these changes should go on top of the ones Farhan has made since this branch was created from that branch.

### CODE REVIEW

File: index_old.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (sstadnicki) [inline index_old.html]: This file shouldn't be needed — that's a big part of the purpose of source control, after all!

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+rect {
```
Comment (sstadnicki) [inline styles.css]: I think this is a stray — this file should be in the styles/ directory (Farhan's change moved it there).

**Review by sstadnicki — APPROVED** [review]
You should wait on Farhan before merging, but it looks good to me.

**Comment by farhan-begg** [comment]
Looks good

**Pull request merged**